include/ignition/gazebo/rendering/SceneManager.hh

```diff
@@ -36,6 +36,7 @@
 
 #include <ignition/rendering/RenderTypes.hh>
 
+#include <ignition/gazebo/components/ParticleEmitter.hh>
```

Comment:
I don't see the `ParticleEmitter` component being used in this file, but I do see `msgs::ParticleEmitter` being used. Should we actually be including `ignition/msgs/particle_emitter.pb.h` instead?

Reply:
Done in defd33f6809b8b204ee91426bd48b6eb4c54fc7e